applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -20,68 +20,81 @@
 # Simple patch tests
```

Comment:
yes it is ok, however we need to have tests to be very short, since otherwise they take way too long. already they are too long to run for us to be able to run on traviscl.

i think we should select for the short suite tests that run in no time (all the small tests together <0.5s), and for the nightly suite tests which run in a total runtime of let's say 10s (i mean the total of the tests run in 10s on a fast machine)